### PR TITLE
Show playlist in GUI

### PR DESCRIPTION
**Pull request by daveystruijk**

### CODE REVIEW

File: contextproject-ms3/src/main/java/contextproject/App.java [hunk 1]
```diff
@@ -1,23 +1,26 @@
...
 import javafx.scene.Scene;
 import javafx.stage.Stage;
 
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import contextproject.controllers.CLIController;
+import contextproject.controllers.WindowController;
+import contextproject.loaders.FolderLoader;
+import contextproject.models.Playlist;
```
Comment (Roy-Graafmans) [inline contextproject-ms3/src/main/java/contextproject/App.java]: wrong input order

File: contextproject-ms3/src/main/java/contextproject/controllers/LibraryController.java [hunk 2]
```diff
@@ -3,17 +3,30 @@
 import javafx.collections.FXCollections;
 import javafx.collections.ObservableList;
 import javafx.fxml.FXML;
-import javafx.scene.control.ListView;
+import javafx.scene.control.TableView;
+import contextproject.models.Playlist;
+import contextproject.models.Track;
```
Comment (Roy-Graafmans) [inline contextproject-ms3/src/main/java/contextproject/controllers/LibraryController.java]: wrong input order

File: contextproject-ms3/src/main/java/contextproject/App.java [hunk 3]
```diff
@@ -1,23 +1,26 @@
...
+
 /**
  * Hello world.
  *
  */
 public class App extends Application {
   static Logger log = LogManager.getLogger(App.class.getName());
+  static final String DIRECTORY = "/Users/daveystruijk/Documents/FEESJE/House (Chill)";
```
Comment (Emiel94) [inline contextproject-ms3/src/main/java/contextproject/App.java]: Use File.separator and user.dir to get the users directory. This wil only work on your machine

File: contextproject-ms3/src/main/java/contextproject/controllers/WindowController.java [hunk 4]
```diff
@@ -2,14 +2,15 @@
 
 import javafx.fxml.FXML;
 import javafx.scene.text.Text;
+import contextproject.models.Playlist;
```
Comment (Roy-Graafmans) [inline contextproject-ms3/src/main/java/contextproject/controllers/WindowController.java]: wrong input order

File: contextproject-ms3/src/main/java/contextproject/controllers/LibraryController.java [hunk 5]
```diff
@@ -3,17 +3,30 @@
...
-  public void setItems() {
-    ObservableList<String> items = FXCollections
-        .observableArrayList("One", "Two", "Three", "Nice!");
-    listView.setItems(items);
+  public void update() {
+    ObservableList<Track> items = FXCollections.observableArrayList(library);
+    tableView.setItems(items);
+  }
+  
+  public Playlist getLibrary() {
```
Comment (Emiel94) [inline contextproject-ms3/src/main/java/contextproject/controllers/LibraryController.java]: Javadoc should be added.

File: contextproject-ms3/src/main/java/contextproject/controllers/LibraryController.java [hunk 6]
```diff
@@ -3,17 +3,30 @@
...
+    return this.library;
+  }
+  
+  public void setLibrary(Playlist playlist) {
```
Comment (Emiel94) [inline contextproject-ms3/src/main/java/contextproject/controllers/LibraryController.java]: Missing javadoc.

File: contextproject-ms3/src/main/java/contextproject/controllers/WindowController.java [hunk 7]
```diff
@@ -2,14 +2,15 @@
...
 
 public class WindowController {
   @FXML private Text actiontarget;
   @FXML private LibraryController libraryController;
   @FXML private PlayerControlsController playerControlsController;
   
-  public void doSomething() {
-    libraryController.setItems();
+  public void setLibrary(Playlist playlist) {
```
Comment (Emiel94) [inline contextproject-ms3/src/main/java/contextproject/controllers/WindowController.java]: Again javadoc

**Pull request merged**